cloudferry_devlab/Vagrantfile

```diff
@@ -1,209 +1,162 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby ts=2 sw=2 et sua= inex= :
 
-config_file = File.join(File.dirname(__FILE__), "/config.ini")
+config_file = File.join(File.dirname(__FILE__), '/config.ini')
 options = {}
-File.foreach(config_file) { |line|
-  option, value = line.split("=")
-  option = option.strip()
-  if !value.nil?
-    value = value.strip()
+File.foreach(config_file) do |line|
+  option, value = line.split('=')
+  option = option.strip
+  unless value.nil?
+    value = value.strip
     options[option] = value
   end
-}
+end
```

Comment:
Let's remove config file for devlab and instead hardcode all values like box IP addresses in Vagrantfile, this will simplify the logic in Vagrantfile even further without sacrificing flexibility.